src/ploomber/cli/cloud.py

```diff
@@ -25,6 +25,7 @@
 
 CLOUD_APP_URL = 'ggeheljnx2.execute-api.us-east-1.amazonaws.com'
```

Comment:
I think this should be the api.ploomber.io, right?

Reply:
Yeah it was tested with the direct api